src/runtime/Temporal.h

```diff
@@ -70,22 +71,27 @@ class Temporal : public DerivedObject {
         return m_temporalPlainDateTimePrototype;
     }
 
+    Object* getTemporalPlainYearMonthPrototype() const
+    {
+        return m_temporalPlainYearMonthPrototype;
+    }
+
     bool isTemporalObject() const override
     {
         return true;
     }
 
-    static Value getOptionsObject(ExecutionState& state, const Value& options)
+    static Value getOptionsObject(ExecutionState& state, const Value* options)
     {
-        if (options.isObject()) {
-            return options;
+        if (!options || options->isUndefined()) {
+            return {};
```

Comment:
We commonly use reference (`&`) for Value type argument than pointer.

How about returning with `Value()` to clearly represent the return value?

Reply:
Sometimes the options argument is not present and to not create an unused c++ object i pass a `nullptr` as the options parameter. (zherczeg requested)

I always use `{}` but i will fix it and in the future i will use `Value()`.